de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/CustomTreeCell.java

```diff
@@ -53,6 +57,8 @@ public CustomTreeCell(WorkspaceFileSystem workspaceFileSystem, FileImageProvider
 
 			final Dragboard db = event.getDragboard();
 			boolean success = false;
+			DropMode dropMode = DropMode.COPY;
+			List<File> directories = new ArrayList<File>();

```

Comment:
This file needs more comments around fields and if-statements to make it easier to understand whats going on.
The code in and of itself is fine though.